src/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -22,11 +23,11 @@ public sealed class LicenseManager
     {
         private static readonly object s_selfLock = new object();
 
-        private static volatile LicenseContext s_context;
+        private static LicenseContext s_context;
         private static object s_contextLockHolder;
         private static volatile Hashtable s_providers;
         private static volatile Hashtable s_providerInstances;
-        private static readonly object s_internalSyncObject = new object();
+        private static object s_internalSyncObject = new object();
```

Comment:
I had to remove volatile at line 26 as it loses meaning while passing as ref and compiler gives error on it. In that flow, I wrongly removed the readonly on this field... Reverting.